infra/charts/turing/subcharts/merlin/values.yaml

```diff
@@ -1,5 +1,6 @@
 global:
   merlin: {}
+  mlp: {}
```

Comment:
Adding this for completeness. When installing the Turing chart, the master values file will define this, so it is not a problem. But if trying to dry-run just the Merlin chart, the absence of this field is problematic.